packages/c/cumem/xmake.lua

```diff
@@ -0,0 +1,21 @@
+package("cumem")
+set_kind("library", { headeronly = true })
+set_description("CUDA Memory Management Wrapper with Type Safety")
+
+set_homepage("https://github.com/BinhaoQin/cuMem")
+
+add_urls("https://github.com/BinhaoQin/cuMem.git")
```

Comment:
please add `https://github.com/BinhaoQin/cuMem/archive/refs/tags/$(version).tar.gz` url and it's sha256 to add_versions